core/chains/evm/config/toml/config_test.go

```diff
@@ -0,0 +1,29 @@
+package toml_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/multierr"
+
+	"github.com/smartcontractkit/chainlink/v2/core/chains/evm/config/toml"
+)
+
+func Test_Validate_Defaults(t *testing.T) {
```

Comment:
```suggestion
func TestEVMConfig_ValidateConfig(t *testing.T) {
```

Reply:
Eh maybe still worth mentioning defaults too?
```suggestion
func TestEVMConfig_ValidateConfig_defaults(t *testing.T) {
```